### ISSUE TITLE
Importing or exporting all tables

### ISSUE THREAD

**Issue by jasonbronson** [TablePlus/TablePlus#1633]
It would be a great enhancement to have when you right click on a database on the left pane to have export or import options. Usually I have table plus on another window and I have to drag the mouse to the file menu to access these functions.

**Comment by huyphams** [comment]
I don't quite understand the suggestion. We already have that menu. Maybe you're using the very very old TablePlus build? or the Linux or iOS version?. Please do not remove the issue template questions.
<img width="518" alt="abc" src="https://user-images.githubusercontent.com/3654622/71560936-3bf1c880-2a46-11ea-9068-e28f709c44c8.png">

**Comment by baohx2000** [comment]
Can't right click tables on Linux build...guess that's still WIP

**Comment by jasonbronson** [comment]
I’m using Mac with latest version. I’m talking about easily exporting the entire db into a SQL dump file. Otherwise I have to use backup method which is a zillion clicks seems like overkill honestly for smaller db’s where I need a quick dev dump.

**Comment by huyphams** [comment]
Thank @jasonbronson , have you tried to press Command + A then right-click > Export... TablePlus will export all selected items.

**Comment by Kerfred** [comment]
I agree with @jasonbronson it would be a great enhancement to have the "Import database" and "Export database" on the right click on the left pane. I mean importing and exporting the whole database without having to select all the tables.

**Comment by ginocremer** [comment]
I also agree with @jasonbronson - In SequelPro I could easily define two shortcuts, one for an export, one for an import to do these tasks very quickly. With TablePlus I have to select, do a right click, click on Export, than select SQL (cause - I don't know why - CSV tab is selected first) and finally...I can export. A "Quick Export" button - or even better - a shortcut to quickly dump the entire DB would be helpful (and would allow us to be as fast as with Sequel Pro)

**Comment by alansikora** [comment]
I don't think we have it, no

**Comment by huyphams** [comment]
Hi @ehausen why not? You can do it with export/import or backup/restore
